tools/launch.py

```diff
@@ -64,13 +64,23 @@ def main():
     parser.add_argument('-H', '--hostfile', type=str,
                         help = 'the hostfile of slave machines which will run \
                         the job. Required for ssh and mpi launcher')
+    parser.add_argument('-SH', '--server-hostfile', type=str,
+                        help = 'the hostfile of server machines which will run \
+                        the job. Required for byteps launcher')
     parser.add_argument('--sync-dst-dir', type=str,
                         help = 'if specificed, it will sync the current \
                         directory into slave machines\'s SYNC_DST_DIR if ssh \
                         launcher is used')
     parser.add_argument('--launcher', type=str, default='ssh',
                         choices = ['local', 'ssh', 'mpi', 'sge', 'yarn'],
                         help = 'the launcher to use')
+    bps_group = parser.add_argument_group('byteps-backend')
+    bps_group.add_argument('--byteps-launch', action='store_true',
```

Comment:
Yes, I will.
BTW, I think the better design is to add a choice in `--launcher`. However, I think I should open a new issue for that change, because it may caused some changes to other launcher.